packages/npm/@amazeelabs/jest-preset/package.json

```diff
@@ -10,15 +10,16 @@
     "access": "public"
   },
   "peerDependencies": {
-    "@types/jest": "^27",
-    "jest": "^27"
+    "@types/jest": "^28",
+    "jest": "^28"
   },
   "dependencies": {
     "@swc/core": "^1.2.204",
     "@swc/jest": "^0.2.21",
-    "@testing-library/react": "^12.1.5",
+    "@testing-library/react": "^13.3.0",
     "@tsconfig/recommended": "^1",
     "identity-obj-proxy": "^3.0.0",
+    "jest-environment-jsdom": "^28.1.3",
```

Comment:
This is now served as a separate package.

Not sure if it should be added to `peerDependencies` 🤔 But then we'll need to add it to pretty much every package.